<?xml version="1.0"?>
<launch>

    <arg name="waypoint_separation" default="5.0" />
    <arg name="is_relative" default="true" />
    <arg name="target_x" default="10" />
    <arg name="target_y" default="1.57" />
    <arg name="isfake" default="false" />
    <arg name="isgazebo" default="false" />
    <arg name="iswamv" default="false" />
    <param name="/use_sim_time" value="$(arg isgazebo)"/>

    <node pkg="robotx_nav" type="move_base_forward.py" name="forward_behavior" >
        <remap from="odom" to="odometry/filtered/global" if="$(arg isgazebo)" />
        <remap from="odom" to="odometry/filtered/global" if="$(arg iswamv)" />
        <remap from="cmd_vel" to="move_base_cmd_vel" if="$(arg iswamv)" />
        <param name="waypoint_separation" value="$(arg waypoint_separation)" />
        <param name="is_relative" value="$(arg is_relative)" />
        <param name="target/x" value="$(arg target_x)" />
        <param name="target/y" value="$(arg target_y)" />
    </node>

</launch>
